### ISSUE TITLE
Feature Request: expose graphviz layout (and other?) options

### ISSUE THREAD

**Issue by sparkofreason** [benedekfazekas/morpheus#1]
Playing with this, the output tends to be very wide and flat. Other graph layouts may improve the visualization, would be nice to at least be able to experiment with them.

**Comment by benedekfazekas** [comment]
experimenting with something else atm on morpehus but quite easy to add this option and feed into `loom` that morpheus uses. happy to merge a PR if you want to get down to it ;)

also if you generate a dot file and upload it to edotor.net you can change the alg/engine I think -- as a quick work around

hope this helps

**Comment by sparkofreason** [comment]
Yes, thanks, using edotor does help. If I can scrape together some time, I'll look into a PR.

The navigable SVG used here is also handy: https://github.com/SevereOverfl0w/vizns

**Comment by benedekfazekas** [comment]
> The navigable SVG used here is also handy: https://github.com/SevereOverfl0w/vizns

looking exactly at that :)

**Comment by benedekfazekas** [comment]
SVGs are navigable now. slightly differently as in `vizns` in that you get to a subgraph where the clicked node is the rootnode when you navigate.

i might add an option that generates links to graphs showing all dependants of a given node alternatively
